docs/team/beefham.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: page
+title: Wee Han's Project Portfolio Page
+---
+
+### Project: HeRon
+
+HeRon is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```

Comment:
Change the rest of the description so that it fits our application more.
e.g. HeRon is a desktop address book application for human resource managers to ...

Reply:
Good catch, I have changed it to match the description used by Sze Long. Thanks!